src/HelpersServiceProvider.php

```diff
@@ -66,20 +66,30 @@ protected function extendValidator(): void
             return !$result;
         });
 
-        Validator::extend('list_exists', function ($attribute, $value, $parameters) {
+        Validator::extend('list_exists', function ($attribute, $value, $parameters, $validator) {
             if (count($parameters) < 1) {
+                $validator->errors()->add($attribute, 'You must add at least 1 parameter');
```

Comment:
I've checked it one more time and it's not good to fail validation in case the developer use validation rule incorrectly. Let's implement the separate `InvalidValidationRuleUsageException` for such cases